queue_services/account-mailer/src/account_mailer/email_templates/business_invitation_email_for_bceid.html

```diff
@@ -26,7 +26,12 @@ <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verd
                             You have been invited to join the <strong>{{ account_name }}</strong> account at Business Registry.
                         </h1>
                         <p style="margin-top: 25px; margin-bottom: 0; line-height: 24px; font-family: Verdana, Arial, sans-serif; font-size: 16px;">
-                            <strong>{{ user_first_name }} {{ user_last_name }}</strong> has invited you to join the <strong>{{ account_name }}</strong> account at Business Registry. You will need to log in using your BCEID to join this account.
+                            <strong>{{ user_first_name }} {{ user_last_name }}</strong> has invited you to join the <strong>{{ account_name }}</strong> account at Business Registry.
+                            {% if role == 'ADMIN' %}
+                            <span>You will need to log in with a registered BCeID account, and verify your identity by a notarized affidavit. After submitting your information, staff will review your affidavit. If approved you will be able to join the account.</span>
+                            {% else %}
+                            <span>You will need to log in using your BCEID to join this account.</span>
+                            {% endif %}
```

Comment:
will this become true for other admin invitations? like bcsc

Reply:
This template is used for only BCeID invitations